Banking/build.gradle

```diff
@@ -15,7 +15,10 @@ repositories {
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
-	runtimeOnly 'mysql:mysql-connector-java'
+    implementation 'org.jetbrains:annotations:19.0.0'
```

Comment:
Follow proper indentations. Best practice is not to hardcode the versions and to have the versions as variables in a separate file, lets say depVersions.gradle which only contains variables representing the version numbers of the dependencies.

Reply:
Sure. I will rectify it.